fix: Mark checkout id as deprecated_required_to_omit for update requests (#145)
Universal-Commerce-Protocol/ucp: docs/playground.md

  }

  prepareUpdatePayload(checkoutResponse) {

>     const patch = { id: checkoutResponse.id }; // deprecated: id is redundant with URL path

Reviewer: ```suggestion
    const patch = { id: checkoutResponse.id }; // deprecated: id is provided in URL path
```
